CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing guidelines
+
+## Prerequisites
+- [go](https://golang.org/dl/) version v1.13+.
+- [docker](https://docs.docker.com/install/) version 17.03+.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.14.1+.
+- [kubebuilder](https://book.kubebuilder.io/quick-start.html#installation) v2.0.1+
+
+## How to run unit tests locally
+1. Make sure kubebuilder is installed.
+2. Run `make test` in the project root directory.
+
+## Developing guidelines
+1. If CRDs in `deploy/crds` are updated, the same file names located in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+2. If `role.yaml`, `role_binding.yaml`, or `operator.yaml` in `deploy` are updated, the ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+3. If `README.md` is updated, ClusterServiceVersion(CSV) file in `deploy/olm-catalog/ibm-block-csi-operator` must be updated accordingly.
+4. Be sure to run `make update` or `make test` before you finish a commit.
```

Comment:
Can this be as a note for step #3? I don't think this should be a separate step

Reply:
But it is not related to #3, and all of them are tips, not a step by step thing, what do you think?